src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependencyTreeFlags.cs

```diff
@@ -88,33 +91,44 @@ public static class DependencyTreeFlags
                 + GenericDependency;
 
         internal static readonly ProjectTreeFlags TargetNode = ProjectTreeFlags.Create("TargetNode");
-        internal static readonly ProjectTreeFlags SubTreeRootNode = ProjectTreeFlags.Create(ProjectTreeFlags.Common.VirtualFolder) +
-                                                                    ProjectTreeFlags.Create(ProjectTreeFlags.Common.ReferencesFolder);
+        
+        internal static readonly ProjectTreeFlags SubTreeRootNode = ProjectTreeFlags.Create("SubTreeRootNode");
 
+        internal static readonly ProjectTreeFlags SubTreeRootNodeFlags
+                = ProjectTreeFlags.Create(ProjectTreeFlags.Common.VirtualFolder)
+                + SubTreeRootNode;
 
+        internal static readonly ProjectTreeFlags AnalyzerSubTreeRootNode = ProjectTreeFlags.Create("AnalyzerSubTreeRootNode");
```

Comment:
Why do these have a different name "SubTreeRootNode" to what you named the context menus? Can we consistently use "GROUP"?

Reply:
"Root" has a pre-existing meaning in the existing menus/groups, and it's not a good fit for the grouping nodes anyway. I considered updating the flags to reflect this, but the existing flags are public API.

Reply:
How are they existing public API? You just just added them.

Reply:
`"SubTreeRootNode"`, from which the naming pattern extends, already existed.

Also, these flags are identical to those recently removed in #5538, before which they were present in many releases with these names.

I prefer the name group and would like to use it consistently, but favoured back compat here. Do you feel it's worth the break here?